gc-debug-specs.adoc

```diff
@@ -0,0 +1,83 @@
+= GC Debug frontend spec
```

Comment:
As a contributor I don't understand this document you added at the root of the project.

Maybe relocating it to a better location and better explaining what this is can help me to better understand this, or better living with ignoring it.

Reply:
I removed those two files. They don't belong to this PR indeed.